.env.local.sample

```diff
@@ -8,3 +8,8 @@ SLACK_API_SECRET=
 SLACK_REDIRECT_URI=https://mojotech-helios2.herokuapp.com/web_hooks/create_slack_auth
 SLACK_VERIFICATION_TOKEN=
 SLACK_WEBHOOK_URL=
+GITHUB_WEBHOOK_SECRET=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+TWITTER_ACCESS_TOKEN=
+TWITTER_ACCESS_SECRET=
```

Comment:
I believe that this change you made fixes an issue I was running into earlier where I tried to strip whitespace from ENV variables that didn't exist. I may drop the check ENV.key?(key) from my PR if you merge this PR, so I will be following this (or I may not drop it because the check doesn't hinder proper functioning of helios, not sure yet).